src/test/java/org/truffleruby/MiscTest.java

```diff
@@ -87,7 +88,9 @@ public void testEvalFromIntegratorThreadSingleThreaded() throws InterruptedExcep
 
     @Test
     public void testFiberFromIntegratorThread() throws InterruptedException {
-        try (Context context = Context.newBuilder().allowCreateThread(true).build()) {
+        try (Context context = Context.newBuilder()
+                .option(OptionsCatalog.SINGLE_THREADED.getName(), Boolean.FALSE.toString())
```

Comment:
It's odd this test didn't set the exit status to failure:
```
  testFiberFromIntegratorThread: Exception in thread "Thread-3" org.graalvm.polyglot.PolyglotException: org.truffleruby.language.control.RaiseException: internal implementation error - threads not allowed in single-threaded mode (RubyTruffleError)
Original Internal Error: 
org.truffleruby.language.control.RaiseException: internal implementation error - threads not allowed in single-threaded mode (RubyTruffleError)
Passed
```

Seems because Thread#join in Java doesn't propagate the exception, + JUnit doesn't complain when there is no assertion in the test it seems :disappointed: